facebook-instant-articles.php

```diff
@@ -404,4 +404,44 @@ function ia_markup_version( ) {
 	add_action( 'wp', 'ia_markup_version' );
 
 	Instant_Articles_Wizard::init();
+
+	function rescrape_article( $post_id, $post ) {
```

Comment:
It is not required, but these are the parameters passed to the hook by WordPress.